src/driver/sqlite/SqliteDriver.ts

```diff
@@ -7,6 +7,9 @@ import {SqliteConnectionOptions} from "./SqliteConnectionOptions";
 import {ColumnType} from "../types/ColumnTypes";
 import {QueryRunner} from "../../query-runner/QueryRunner";
 import {AbstractSqliteDriver} from "../sqlite-abstract/AbstractSqliteDriver";
+import {close, open, stat} from "fs";
+import {dirname} from "path";
+import * as mkdirp from "mkdirp";
```

Comment:
we cannot use nodejs dependencies because typeorm is crossplatform (there is no node deps in browser for example). extract "fs" and "path" logic into PlatformTools and add corresponding code into BrowserPlatformTools. Also probably its better to write recursive directory creation code manually then adding another dependency...

Reply:
Looks like we already have `mkdirp` package, so you only need to abstract things into PlatformTools